sdk/appconfiguration/Azure.ApplicationModel.Configuration/src/Azure.ApplicationModel.Configuration.csproj

```diff
@@ -10,16 +10,9 @@
       ]]>
     </PackageReleaseNotes>
 
-    <!-- Make sure that we don't pull in additional dependencies during build or package -->
-    <ImportDefaultReferences>false</ImportDefaultReferences>
-
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
     <NoWarn>$(NoWarn);3021</NoWarn>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <AssemblyName>Azure.ApplicationModel.Configuration</AssemblyName>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <UseProjectReferenceToAzureCore>true</UseProjectReferenceToAzureCore>
```

Comment:
why is it that we are removing the reference to Azure.Core from the projects?

Reply:
The reference still exists on line 31.   The decision to toggle between the project and package reference was hoisted to the `eng/Directory.Build.Data.props` file and defaulted to `true`.    

Since Configuration wasn't setting a different value, the override isn't needed any longer.   :)

Reply:
I missed that one in the .props file. Thanks Jesse!